src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
+    public int getRandomNumber() {
+        return new Random().nextInt(100);
```

Comment:
Magic number 100 detected. It's better to declare it as a constant field to increase the readability and maintainability of your code. For example, private static final int MAX_NUMBER = 100; and then use MAX_NUMBER in the nextInt() call.